content/docs/iac/using-pulumi/continuous-delivery/gitlab-app.md

```diff
@@ -16,28 +16,63 @@ aliases:
 - /docs/iac/packages-and-automation/continuous-delivery/gitlab-app/
 ---
 
-With this GitLab integration, Pulumi is able to add summary notes to a GitLab Merge Request by using the merge request information
-posted to the Pulumi Cloud via [GitLab Webhooks](https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#merge-request-events).
+With this GitLab integration, Pulumi is able to add Pulumi Previews to a GitLab Merge Request.
 
-To enable the integration with your GitLab project, you will need to ensure you have done the following two things:
+## Integration Methods
 
-* [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
-* If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)
+There are two ways to integrate Pulumi and GitLab.
 
-## Prerequisites
+* If you are a Premium or Ultimate GitLab customer, you will use a GitLab Group Access Token. This integration type is simpler to setup and is not tied to a specific user.
+  * The user configuring the integration will need to be a GitLab Organization Owner.
```

Comment:
i think we can just add this to previous bullet as a second sentence (style guide: never have a list of one)

while i am typing ... is this for DOC as opposed to BLOG?  (why do i see your amazing picture below?)